### DISCUSSION TITLE
Can we add custom footer for table-repeater?

### DISCUSSION THREAD

**Discussion by lakuapik** [awcodes/filament-table-repeater#35] [Q&A] [answered]
In the filamentphp default table, there is a way to add custom footer for the table. like this one mentioned here https://github.com/filamentphp/filament/discussions/5146. 

<img width="1313" alt="image" src="https://user-images.githubusercontent.com/20186786/230761646-b08b613f-b240-4901-bc61-bc6b4240f001.png">

I was curious, is there a way to do that with table-repeater?

Here is the repeater table, i want to sum the subtotal.

<img width="1251" alt="image" src="https://user-images.githubusercontent.com/20186786/230761732-56f12e77-712b-41a0-a720-3a7e03b05165.png">

Also thank you for this amazing extension!

**Comment by awcodes** [comment, marked as answer]
Hmm. Let me think about this. It's not really the purpose of this plug in, but I can see a use case for it. 

I just don't want to get into the territory of recreating the actual Tables package.

**Reply by lakuapik** [reply]
I see. @awcodes thank you for the answer. Right now i managed to get it done by appending the sum data into the array itself.

<img width="1261" alt="image" src="https://user-images.githubusercontent.com/20186786/231002239-4d02f130-83a3-461b-85d3-1f13a9479921.png">

It is possible to do this, but just not as flexible as custom footer for filament table.

**Reply by awcodes** [reply]
I hear you. Glad you found a solution though.
